### ISSUE TITLE
provide signals before apply as pymongo(maybe more libs) need close before fork 

### ISSUE THREAD

**Issue by zjqzero** [nalepae/pandarallel#114]
This is not an issue. If not necessary, just close.  
thanks.

![image](https://user-images.githubusercontent.com/12052416/96407931-0b695e00-1215-11eb-878d-1a7ffacd969f.png)

**Comment by shermansiu** [comment]
Yeah, this is something that needs to be dealt with in general when using multiprocessing with Pymongo.

This is out of scope for Pandarallel and I'd prefer to close this issue.
